domains/codexprt.json

```
@@ -0,0 +1,11 @@
+{
+    "domain":"codexprt.is-a.dev",
+    "subdomain":"codexprt",
+    "owner":{
+        "email":"yourwebcoderr@gmail.com"
+    },
```

you need to add your github username, please read [the docs](https://docs.is-a.dev/domain-structure/#owner-required).
```suggestion
    "owner": {
        "username": "CodeCurio",
        "email": "yourwebcoderr@gmail.com"
    },
```